japanese/garden/garden.md

```diff
@@ -1,6 +1,8 @@
-何が起きているのかを確かめるために、とりあえず庭に出てみた。
+﻿何が起きているのかを確かめるために、とりあえず庭に出てみた。
 地面を覆う苔と、手の行き届いた木々がきれいな庭だ。
 ここからは、他の建物は見えない。
 林の向こうのほうで川の流れる音が聞こえる。
 
-さてあなたは：
\ No newline at end of file
+さてあなたは：
+
+[林の中に分け入る](../forest/beanus)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.